android/java/org/chromium/chrome/browser/crypto_wallet/util/ParsedTransaction.java

```diff
@@ -251,6 +252,11 @@ public static ParsedTransaction parseTransaction(
                         new SolanaInstructionPresenter(solanaInstruction);
                 String lamport = presenter.getLamportAmount();
                 Integer instructionType = presenter.getInstructionType();
+                if (instructionType != null
+                        && (instructionType == SolanaSystemInstruction.ASSIGN_WITH_SEED
+                                || instructionType == SolanaSystemInstruction.ASSIGN)) {
+                    parsedTransaction.solChangeOfOwnership = true;
+                }
```

Comment:
Adopting the same logic as Desktop https://github.com/brave/brave-core/blob/833b5a634bf17e544f196cf6ec3573997c656f5f/components/brave_wallet_ui/utils/tx-utils.ts#L1834